tests/cross_validation/test_lal_overlap.py

```
@@ -66,24 +66,24 @@
 # implementations; simpler/more-analytical waveforms achieve near-machine
 # precision while complex ones accumulate more rounding error.
 #
 # IMRPhenomPv2 NOTE: the 1e-4 threshold is intentionally loose and does NOT
-# reflect a deficiency in ripple's implementation. The mismatch is entirely a
-# pure time shift (amplitude agreement is perfect to <1e-9). The shift arises
-# because LAL computes the coalescence-time correction t0 via a 10-point
+# reflect a deficiency in ripple's implementation. The overlap loss is entirely
+# due to a pure time shift (amplitude agreement is perfect to <1e-9). The shift
+# arises because LAL computes the coalescence-time correction t0 via a 10-point
 # natural cubic spline over [0.8*f_RD, 1.2*f_RD] (GSL gsl_interp_cspline,
 # LALSimIMRPhenomP.c lines 1060–1151), whereas ripple uses exact JAX autodiff.
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)